### ISSUE TITLE
PowerDNS 3.4.6 bind-backend chokes on NULL MX

### ISSUE THREAD

**Issue by pieterlexis** [PowerDNS/pdns#2836]
This line in a zonefile:

```
@ IN MX 0 .
```

fails with:
```
Oct 27 08:23:57 pdns-public-ns1 pdns_server[551]: Oct 27 12:23:57 Signing thread died because of std::exception: Parsing record content: expected digits at position 0 in '.'
Oct 27 08:23:57 pdns-public-ns1 pdns[551]: Signing thread died because of std::exception: Parsing record content: expected digits at position 0 in '.'
```

This is valid according to [RFC 7505](https://tools.ietf.org/rfc/rfc7505.txt).

**Comment by pieterlexis** [comment]
this is not an issue in the current master.

**Comment by pieterlexis** [comment]
This only happens on the 3.4.1-X on debian jessie and was fixed in our 3.4.2 with e94c2c40ac6e0ecf8a5fdf7c415da5c59a8330d6
